specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/applicationGateway.json

```
@@ -1589,6 +1589,13 @@
         "provisioningState": {
           "type": "string",
           "description": "Provisioning state of the backend http settings resource. Possible values are: 'Updating', 'Deleting', and 'Failed'."
+        },
+         "port": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Custom port which will be used for probing the backend servers. The valid value ranges from 1 to 65535. In case not set, port from http settings will be used. This property is valid for Standard_v2 and WAF_v2 only."
+          "minimum": 1
```

Missing trailing comma
```suggestion
          "minimum": 1,
```